plugins/asterisk/asterisk_18_fax_spandsp/asterisk_fax_negotiations_failed

```diff
@@ -124,8 +124,8 @@ address with the copyright notice upgrade with your name.
     $astman->disconnect;
 
     # Some idiot who wrote the SpanDSP code spelled "Negotiation" wrong.
-    print "t38_failed.value $faxstats{'Spandsp T.38'}{'Negotation Failed'}\n";
-    print "g711_failed.value $faxstats{'Spandsp G.711'}{'Negotation Failed'}\n";
+    print "t38_failed.value $faxstats{'Spandsp T.38'}{'Negotiation Failed'}\n";
+    print "g711_failed.value $faxstats{'Spandsp G.711'}{'Negotiation Failed'}\n";
```

Comment:
These two lines should not be changed.  The upstream source seem to output this typo.

https://gitorious.org/asterisk-tools/asterisk/source/6a47bf7361723b82e7e45465f257798218ed7f21:res/res_fax_spandsp.c#L709